### ISSUE TITLE
Scaffolding (on 1.0 project) a view with model and without dbContext fails for Delete, Details and List templates

### ISSUE THREAD

**Issue by prafullbhosale** [aspnet/Scaffolding#425]
Repro Steps: 

1. Create a new Asp.Net Core project with Individual auth (1.0)
2. Add a model class 
```cs
public class Car 
{
   public int ID { get; set;}
   public class Name { get; set;}
}
```
3. Right click Views folder -> Add View

Scaffolding fails with below error
![image](https://cloud.githubusercontent.com/assets/1521254/24259012/c35becae-0fac-11e7-849c-38adfdd1fd88.png)

**Comment by Eilon** [comment]
This patch fix is *approved*. Please follow the normal code review / pull request process and make sure you make the change in the correct branch.

**Comment by prafullbhosale** [comment]
I am assuming the other issue for 1.1 (https://github.com/aspnet/Scaffolding/issues/427) is also approved.

**Comment by Eilon** [comment]
@prafullbhosale I will approve it shortly; just waiting for the build system to be up and running for the patch.

**Comment by Eilon** [comment]
@prafullbhosale / @mlorbetske - do you guys have a status update on this bug? We want to get the patches done ASAP.

**Comment by prafullbhosale** [comment]
The branches are setup and included in the `Coherence-Patch` builds.
Only remaining part is to bump up the versions for latest packages. Have those on PR https://github.com/aspnet/Scaffolding/pull/453 and https://github.com/aspnet/Scaffolding/pull/452

**Comment by prafullbhosale** [comment]
Fixed in `rel/tools-1.0.1`

**Comment by Eilon** [comment]
Preview builds of this patch fix should be available on the following feeds:

* https://dotnet.myget.org/gallery/aspnet-1-0-5-may2017-patch-public (for fixes in the 1.0.x release)
  * Feed URL to add to NuGet.config: https://dotnet.myget.org/F/aspnet-1-0-5-may2017-patch-public/api/v3/index.json
* https://dotnet.myget.org/gallery/aspnet-1-1-2-may2017-patch-public (for fixes in the 1.1.x release)
  * Feed URL to add to NuGet.config: https://dotnet.myget.org/F/aspnet-1-1-2-may2017-patch-public/api/v3/index.json

If you have a chance, please try out these preview builds and let us know if you have any feedback!

**Comment by prafullbhosale** [comment]
Tested the scaffolding packages from the feed, the fix looks good.
